runtime/vscode/statusline.lua

```diff
@@ -3,10 +3,12 @@ local api = vim.api
 local ns = api.nvim_create_namespace("vscode.statusline")
 local curr_status = ""
 
+local DEFAULT_STATUSLINE = api.nvim_get_option_info2("statusline", {}).default
+
 local function refresh()
   local status = ""
 
-  if vim.o.laststatus == 0 or vim.o.statusline == "" then
+  if vim.o.laststatus == 0 or vim.o.statusline == "" or vim.o.statusline == DEFAULT_STATUSLINE then
```

Comment:
isn't there an early init area where vscode-neovim sets default options? then the user can override that if they want, or augment it with `+=`.

Reply:
somehow the latest push is even more complicated.

my suggestion was simply to set a default 'statusline' *early*, wherever vscode-neovim sets its defaults. we don't need to do backflips to override anything that happens after that.